fusor/fusor.py

```diff
@@ -254,7 +253,9 @@ async def transcript_segment_element(
         :return: Transcript Segment Element, warning
         """
         if tx_to_genomic_coords:
-            data = await self.cool_seq_tool.transcript_to_genomic_coordinates(**kwargs)
+            data = await self.cool_seq_tool.ex_g_coords_mapper.transcript_to_genomic_coordinates(
```

Comment:
possibly not the intended use, but I tried to simply use CST's `transcript_to_genomic_coordinates` from `default.py` but kept running into issues/errors, but this worked. lmk if I should keep trying the other way though

Reply:
this is correct